Application.mk

```diff
@@ -93,6 +93,21 @@ ifneq ($(BUILD_MODULE),y)
   OBJS += $(MAINCOBJ) $(MAINCXXOBJ) $(MAINRUSTOBJ) $(MAINZIGOBJ)
 endif
 
+ifneq ($(PROGNAME),)
+  PROGOBJ := $(MAINCOBJ) $(MAINCXXOBJ) $(MAINRUSTOBJ)
+  PROGNAMELIST := $(PROGNAME)
```

Comment:
why not use PROGNAME directly?

Reply:
> why not use PROGNAME directly?

**Removed** 
this is because the elf mode and the builtin mode were not separated before.
`PROGNAME` for elf , `PROGNAMELIST` for builtin.
has now been deleted at this time.